src/scripts/controller/productController.ts

```diff
@@ -0,0 +1,73 @@
+import { Product } from '../model/interfaces';
+import { debounce } from '../utils/debounce';
+import { ProductView } from '../view/productView';
+import { ApiServices } from './apiServices';
+
+export class ProductControler {
```

Comment:
Nit
Did you mean `ProductController` ?
```suggestion
export class ProductController {
```

Reply:
Renamed function names (https://github.com/krushna-simform/product-management-system/pull/3/commits/7007557a824ca530bdaa5a53df8427b0966f6a0b)